runtime/src/accounts_db.rs

```diff
@@ -6629,6 +6629,51 @@ pub mod tests {
         assert_eq!(accounts.alive_account_count_in_slot(1), 0);
     }
 
+    #[test]
+    fn test_clean_multiple_zero_lamport_store_matches_index() {
```

Comment:
nit: update test name to `test_clean_multiple_zero_lamport_decrements_index_ref_count()`